src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -348,7 +349,7 @@ private void EvictLocked()
                 {
                     if ((this.random.Next() % 3) == 0)
                     {
-                        this.logger.LogTrace("Transaction ID '{0}' selected to be removed from the cache.", entry.Key);
+                        this.logger.LogTrace("Transaction Id '{0}' selected to be removed from the cache, CacheItem={1}", entry.Key, entry.Value.UnspentOutputs);
```

Comment:
entry.Value.UnspentOutputs could be null no?

Reply:
`entry.Value` will always a `CacheItem` instance? I dont think we ever insert a null `CacheItem` into the dictionary?